pkg/defaultmonitortests/types.go

```diff
@@ -180,6 +181,8 @@ func newUniversalMonitorTests(info monitortestframework.MonitorTestInitializatio
 	monitorTestRegistry.AddMonitorTestOrDie("e2e-test-analyzer", "Test Framework", e2etestanalyzer.NewAnalyzer())
 	monitorTestRegistry.AddMonitorTestOrDie("event-collector", "Test Framework", watchevents.NewEventWatcher())
 	monitorTestRegistry.AddMonitorTestOrDie("clusteroperator-collector", "Test Framework", watchclusteroperators.NewOperatorWatcher())
+	monitorTestRegistry.AddMonitorTestOrDie("initial-and-final-operator-log-scraper", "Test Framework", operatorloganalyzer.InitialAndFinalOperatorLogScraper())
+	monitorTestRegistry.AddMonitorTestOrDie("lease-checker", "Test Framework", operatorloganalyzer.OperatorLeaseCheck())
```

Comment:
I'm wondering why this doesn't show up as an availability issue? After all, not being able to renew the lease is an availability issue

Reply:
> I'm wondering why this doesn't show up as an availability issue? After all, not being able to renew the lease is an availability issue

Availability is checked at a particular time. During install, openshift-tests is not running.  This check works by pulling the operator logs, so it can inspect failures from before openshift-tests started running.